.github/workflows/pr_stackablectl.yml

```diff
@@ -65,3 +65,12 @@ jobs:
       - name: Build for non-Windows
         if: matrix.os != 'windows-latest'
         run: cargo build --target ${{ matrix.target }} -p stackablectl
+
+      - name: Ensure shell completions up-to-date
+        if: matrix.os = 'x86_64-unknown-linux-gnu'
+        run: |
+          ./target/${{ matrix.target }}/debug/stackablectl completions bash > extra/completions/stackablectl.bash
+          ./target/${{ matrix.target }}/debug/stackablectl completions fish > extra/completions/stackablectl.fish
+          ./target/${{ matrix.target }}/debug/stackablectl completions zsh > extra/completions/_stackablectl
```

Comment:
There is an xtask to generate completions: `cargo xtask gen-comp`

Reply:
Nice! I'll run that instead, then do the diff

Reply:
I see there is also a gen-man task, but it doesn't change. I'll leave it out for now.

Reply:
Fixed in 80b9128

Reply:
> I see there is also a gen-man task, but it doesn't change. I'll leave it out for now.

This usually only changes for top-level command changes or when we bump the version. It might, however, still be a good idea to also check that via the workflow. I let you decide if you want to include it in this PR or not.

Reply:
Cool, I might as well add it now, but as a separate step in the job.